databaseconnect.py

```diff
@@ -163,42 +162,43 @@ def learn_question_response(H):
     cur.execute("SELECT id FROM statement_table ORDER BY id DESC")
     res = cur.fetchone()
     last_id = res[0]
-    cur.execute(f"UPDATE statement_table SET sentence={H} WHERE id={last_id}")
+    #cur.execute("UPDATE statement_table SET sentence={H} WHERE id={last_id}")
+    cur.execute("UPDATE statement_table SET sentence = %s WHERE id = %s",(H, last_id))
```

Comment:
Do away with f strings. Remove comment, make changes as discussed.

Reply:
Hi @vishakha-lall , I did some blunder while pushing the changes. I was about to mention that, I did this sticking to the same practise as implemented in `INSERT` queries and to prevent the vulnerability caused due to formatted strings.

Reply:
> Do away with f strings. Remove comment, make changes as discussed.

Yes, sure.